packages/evm/test/registry/guardians-registry-staking.js

```diff
@@ -702,6 +702,223 @@ contract('GuardiansRegistry', ([_, guardian, anotherGuardian]) => {
         itHandlesStakesProperlyForDifferentRecipients(data)
       })
     })
+
+    context('when the guardian requests to activate the tokens and lock activation', () => {
+      const data = LOCK_ACTIVATION_DATA
+
+      beforeEach('allow lock manager', async () => {
+        await registry.changeLockManager(guardian, true, { from: governor })
+      })
+
+      const itHandlesStakesWithLockActivationProperlyFor = (recipient, amount, data) => {
+        it('creates the lock', async () => {
+          await registry.stakeFor(recipient, amount, data, { from })
+
+          const { amount: lockedAmount, total } = await registry.getActivationLock(recipient, guardian)
+          assertBn(lockedAmount, amount, 'locked amount does not match')
+          assertBn(total, amount, 'total locked amount does not match')
```

Comment:
We should also check the lock manager

Reply:
Note that I'm using the guardian as the lock manager, is part of the query